daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/unparsers/runtime1/PackedBinaryUnparserTraits.scala

```diff
@@ -94,9 +94,9 @@ abstract class PackedBinaryDecimalBaseUnparser(
     val bigDec = number.asInstanceOf[JBigDecimal]
     if (bigDec.movePointRight(binaryDecimalVirtualPoint).scale != 0) {
       e.schemaDefinitionError(
-        "Decimal point of number '%s' does not match the binaryVirtualDecmialPoint: %d",
+        "Decimal point of number '%s' does not match the binaryVirtualDecmialPoint: %s",
         bigDec,
-        binaryDecimalVirtualPoint,
+        binaryDecimalVirtualPoint.toString,
```

Comment:
This is a java-ism about varargs. Can't pass primitive types from scala code to java varargs code.